src/agent/src/netlink.rs

```diff
@@ -450,6 +640,17 @@ impl Handle {
         Ok(())
     }
 
+    async fn delete_rules<I>(&mut self, rules: I) -> Result<()>
+    where
+        I: IntoIterator<Item = packet::RuleMessage>,
+    {
+        for rule in rules.into_iter() {
+            self.handle.rule().del(rule).execute().await?;
```

Comment:
Maybe it's better to print some message when delete operation.

self.handle.rule().del(rule).execute().await.context("")?;

Reply:
Thx.  The msg for delete rules has been added where the function is called.